packages/lambda-tiler/README.md

```diff
@@ -1,3 +1,130 @@
 # @basemaps/lambda-tiler
 
-XYZ interface for map generation
+Lambda server for xyz map generation to provide apis for serving both raster and vector map.
+
+## APIs
+
+`Host: https://basemaps.linz.govt.nz/`
+
+### Server APIs
+
+Validate the server status, health, version which is using for the ci/cd.
+
+```typescript
+handler.router.get('/ping', pingGet);
```

Comment:
We shouldnt need to duplicate this code here, could link the `/ping` to the file that implmenets the ping?

# Health endpoints
- [/ping](./src/.../ping.ts)